backend/services/tests/neurons/financial/test_symbols.py

```diff
@@ -40,15 +40,15 @@ class TestWhenCountryIsGiven:
         def test_it_should_narrow_down_based_on_the_market(self, market_find_all):
             config = symbols.Config(country="United States of America")
             market_find_all.return_value = ["NASDAQ"]
-            symbols.run(config)
+            list(symbols.run(config))
```

Comment:
Since this is an iterator now, you have to do `list(...)` or nothing actually happens